pkg/operator/admission/admission_controller.go

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
nit: You're already in the admission folder, so you can call the file controller.go. Single word filenames are preferred if possible.